src/sentry/middleware/devtoolbar.py

```diff
@@ -45,7 +45,12 @@ def _record_api_request(request: HttpRequest, response: HttpResponse) -> None:
     project_id, project_slug = parse_id_or_slug_param(project_id_or_slug)
 
     origin = origin_from_request(request)
-    query_string: str = get_query_string(request)  # starts with '?'
+    query_string = get_query_string(request) or None
+    if query_string:
+        params = query_string.lstrip("?").split("&")
+        params = list(filter(lambda s: s != "queryReferrer=devtoolbar", params))
+        query_string = f"?{'&'.join(params)}" if params else None
+
```

Comment:
what's going on here?

Reply:
From description:

> For SQL query convenience, the analytics event will exclude this from the query string.

It finds the "queryReferrer=devtoolbar" in the query string and scrubs it out

Reply:
Why do we need that for query convenience? What will queries look like with or without it?

Reply:
Hm ok thinking about it now, I don't think there's a guarantee on the positioning of params, so our queries would look like `WHERE query_string LIKE '%key=val%'` regardless. So this doesn't make queries more convenient.

This does reduce the amount of data stored though. Lmk if you want to keep it or not, I don't have a preference anymore

Reply:
we could chop it out and keep the code simple. if we need something like that later it can be added back in.